apis/schema/v1alpha1/mongodbdatabase_webhook.go

```diff
@@ -55,7 +55,7 @@ func (in *MongoDBDatabase) ValidateCreate() error {
 	var allErrs field.ErrorList
 	path := field.NewPath("spec")
 	if in.Spec.Init != nil && in.Spec.Init.Initialized {
-		allErrs = append(allErrs, field.Invalid(path.Child("init").Child("initialized"), in.Name, `cannot set the initialized field to true directly`))
+		allErrs = append(allErrs, field.Invalid(path.Child("init").Child("initialized"), in.Name, MongoDBValidateInitializedSetError))
 		return apierrors.NewInvalid(in.GroupVersionKind().GroupKind(), in.Name, allErrs)
 	}
```

Comment:
This will not work, Create doesn't mean the object is just created.

Reply:
So, How can I ensure, so that .. Init.Initialized field will not be set when creation ?  Is there any other way without ValidateCreate() to do so ?

Reply:
Actually, we can't ensure that. In MongoDB it works similarly, we don't ensure that it is not set.